### DISCUSSION TITLE
Круговая диаграмма для кластеризатора

### DISCUSSION THREAD

**Discussion by YaredFall** [yandex-maps-unofficial/vue-yandex-maps#364] [Q&A (1.x)] [answered]
В текущий момент в документации не упоминается возможность использования круговых диаграмм в качестве маркеров кластеризатора.
Мне не удается найти параметры, отвечающие за отрисовку диаграмм. Есть ли вообще такая возможность, и как её реализовать?

**Comment by daniluk4000** [comment]
https://yandex-maps-unofficial.github.io/vue-yandex-maps/examples/objects/placemark.html

С помощью дефолт маркера, по сути. У него есть такие опции. 

Вы бы хотели использовать их в #cluster, верно? 

Если да, то давайте попробуем завести issue на это. Я ничего не обещаю, но попробую что-то придумать

**Reply by YaredFall** [reply]
> С помощью дефолт маркера, по сути. У него есть такие опции.

Как я вижу в примере, диаграмма строится с помощью кругового градиента через CSS. Мне интересно, нельзя ли воспользоваться готовой схемой отрисовки, как [это было с апи карт второй версии](https://yandex.ru/dev/maps/jsbox/2.1/clusterer_pie_chart).

> Вы бы хотели использовать их в #cluster, верно?

Да, хотелось бы

**Reply by daniluk4000** [reply]
> Как я вижу в примере, диаграмма строится с помощью кругового градиента через CSS. Мне интересно, нельзя ли воспользоваться готовой схемой отрисовки, как [это было с апи карт второй версии](https://yandex.ru/dev/maps/jsbox/2.1/clusterer_pie_chart).

Можно, но не в кластере :)

Они точно рисуются дефолт маркером, это готовая схема

> Да, хотелось бы

Заводите реквест, но скорее всего сделаю только в мажорке v3 =(

**Reply by DangerBadger** [reply]
> Они точно рисуются дефолт маркером, это готовая схема

Добрый день. Интересует тот же вопрос. Не могли бы вы привести пример реализации круговой диаграммы подобным способом?

**Reply by daniluk4000** [reply, marked as answer]
Всем приношу извинения. Сам себя запутал. 

В API Яндекса нет функции рисования диаграм. Только лапками.
